### ISSUE TITLE
Metrics: counter name changed in 1.24.0

### ISSUE THREAD

**Issue by yanns** [apollographql/router#3443]
**Describe the bug**
The name of counters exposed in metrics has changed in [version 1.24.0](https://github.com/apollographql/router/blob/HEAD/CHANGELOG.md#1240---2023-07-13)

The metric `apollo_router_http_requests_total` is now `apollo_router_http_requests_total_total`

**To Reproduce**
Fetch the metrics endpoint scratched by prometheus on http://127.0.0.1:9090/metrics:
```
# HELP apollo_router_http_requests_total_total apollo_router_http_requests_total # TYPE apollo_router_http_requests_total_total counter apollo_router_http_requests_total_total{error="There was no GraphQL operation to execute. Use the `query` parameter to send an operation, using either GET or POST.",service_name="apollo-router",status="400",otel_scope_name="apollo/router",otel_scope_version=""} 1 # HELP apollo_router_processing_time apollo_router_processing_time # TYPE apollo_router_processing_time histogram apollo_router_processing_time_bucket{service_name="apollo-router",otel_scope_name="apollo/router",otel_scope_version="",le="+Inf"} 1 apollo_router_processing_time_sum{service_name="apollo-router",otel_scope_name="apollo/router",otel_scope_version=""} 0.000226153 apollo_router_processing_time_count{service_name="apollo-router",otel_scope_name="apollo/router",otel_scope_version=""} 1 # HELP apollo_router_session_count_active apollo_router_session_count_active # TYPE apollo_router_session_count_active gauge apollo_router_session_count_active{service_name="apollo-router",otel_scope_name="apollo/router",otel_scope_version=""} 0 # HELP apollo_router_session_count_total apollo_router_session_count_total # TYPE apollo_router_session_count_total gauge apollo_router_session_count_total{listener="http://0.0.0.0:4000",service_name="apollo-router",otel_scope_name="apollo/router",otel_scope_version=""} 0 apollo_router_session_count_total{listener="http://0.0.0.0:8088",service_name="apollo-router",otel_scope_name="apollo/router",otel_scope_version=""} 0 apollo_router_session_count_total{listener="http://0.0.0.0:9090",service_name="apollo-router",otel_scope_name="apollo/router",otel_scope_version=""} 1 # HELP apollo_router_span apollo_router_span # TYPE apollo_router_span histogram apollo_router_span_bucket{kind="busy",service_name="apollo-router",span="request",otel_scope_name="apollo/router",otel_scope_version="",le="+Inf"} 1 apollo_router_span_sum{kind="busy",service_name="apollo-router",span="request",otel_scope_name="apollo/router",otel_scope_version=""} 0.00002987 apollo_router_span_count{kind="busy",service_name="apollo-router",span="request",otel_scope_name="apollo/router",otel_scope_version=""} 1 apollo_router_span_bucket{kind="duration",service_name="apollo-router",span="request",otel_scope_name="apollo/router",otel_scope_version="",le="+Inf"} 1 apollo_router_span_sum{kind="duration",service_name="apollo-router",span="request",otel_scope_name="apollo/router",otel_scope_version=""} 0.000083233 apollo_router_span_count{kind="duration",service_name="apollo-router",span="request",otel_scope_name="apollo/router",otel_scope_version=""} 1 apollo_router_span_bucket{kind="idle",service_name="apollo-router",span="request",otel_scope_name="apollo/router",otel_scope_version="",le="+Inf"} 1 apollo_router_span_sum{kind="idle",service_name="apollo-router",span="request",otel_scope_name="apollo/router",otel_scope_version=""} 0.000042837 apollo_router_span_count{kind="idle",service_name="apollo-router",span="request",otel_scope_name="apollo/router",otel_scope_version=""} 1 # HELP otel_scope_info Instrumentation Scope metadata # TYPE otel_scope_info gauge otel_scope_info{otel_scope_name="apollo/router",otel_scope_version=""} 1
```

**Desktop (please complete the following information):**
 - Version 1.24.0

**Additional context**
My quick guess, as I had a similar issue with other libraries in other languages:
With https://github.com/apollographql/router/issues/2878, maybe the library now automatically adds the `_total` suffix to all counters?

**Comment by garypen** [comment]
This was documented in the release notes for 1.24.0: https://github.com/apollographql/router/releases

**Comment by yanns** [comment]
I'm still seeing this issue in v1.25.0

```
test failed. metrics should contain '# TYPE apollo_router_http_requests_total counter'.
Instead the value of metrics is:
checking connectivity to http://127.0.0.1:9090/metrics # HELP apollo_router_http_requests_total_total apollo_router_http_requests_total # TYPE apollo_router_http_requests_total_total counter
```

**Comment by BrynCooke** [comment]
Reopening pending investigation.

**Comment by garypen** [comment]
@yanns I think your comment was intended for #3491 which is fixed and will be released in 1.26.0. There is an alpha release you can test if you are in a hurry: https://github.com/apollographql/router/releases

**Comment by BrynCooke** [comment]
Closing, the metric name is correct, but the description was fixed until #3491
